### PR TITLE
DISTMYSQL-269 Release notes for PDPS 8.0.30 second update

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes-ps-v8.0.30.upd2.md
modified:   docs/release-notes.md
modified:   mkdocs-base.yml

### CODE REVIEW

**Review by eleo007 — APPROVED** [review]

**Review by patrickbirch — APPROVED** [review]
good

File: docs/release-notes-ps-v8.0.30.upd2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+| Release date    | January 23, 2023 |
+| :-------------- | :--------------- |
+|**Installation** | [Installing Percona Distribution for MySQL](installing.md)|
+
+Percona Distribution for MySQL is a single solution with the most critical enterprise components from the MySQL open source community, designed and tested to work together.
+
+Percona Distribution for MySQL provides two deployment variants: one is based on *Percona Server for MySQL* and another one is based on *Percona XtraDB Cluster*. 
+
+This update to the release of Percona Distribution for MySQL using the *Percona Server for MySQL* includes a new version of [Percona Toolkit 3.5.1](https://docs.percona.com/percona-toolkit/release_notes.html#v3-5-1-released-2023-01-13).
```
Comment (nastena1606) [inline docs/release-notes-ps-v8.0.30.upd2.md]: Toolkit release date is 2023-01-13. I believe it should be 2023-01-23

Comment (alina-derkach-oaza) [reply docs/release-notes-ps-v8.0.30.upd2.md]: Fixed

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**